qa/tasks/cephfs/filesystem.py

```diff
@@ -1712,11 +1712,11 @@ def shrink(self, new_max_mds, status=None):
         self.set_max_mds(new_max_mds)
         return self.wait_for_daemons()
 
-    def run_scrub(self, cmd, rank=0):
-        return self.rank_tell(["scrub"] + cmd, rank)
+    def run_scrub(self, cmd, rank=0, timeout=300):
+        return self.rank_tell(["scrub"] + cmd, rank=rank, timeout=timeout)
```

Comment:
since we're only concerned about the scrub status running out of time, how about keeping default time to 120 here? otherwise we might unnecessarily wait for high timeout for all other scrub cmds across fs suite.

Reply:
IMO, that's fine to keep a higher timeout of scrub commands in general. Other commands won't be affected in the majority of cases.